docs/.vitepress/config.js

```diff
@@ -34,7 +34,8 @@ export default defineConfig({
           { text: 'Mosaic vgplot', link: '/vgplot/' },
           { text: 'Mosaic Spec', link: '/spec/' },
           { text: 'Mosaic Server', link: '/server/' },
-          { text: 'Jupyter Widget', link: '/jupyter/' }
+          { text: 'Jupyter Widget', link: '/jupyter/' },
+          { text: 'Developing Clients', link: '/web-frameworks/' }
```

Comment:
Maybe `Web Clients` would be a clearer title than "Developing Clients"? Or `Use in Web Frameworks`?